core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -328,7 +332,8 @@ describe('Creator dashboard controller', () => {
         $rootScope.$apply();
       });
 
-      it('should evaluate data from backend', function() {
+      it('should initialize correctly $scope properties after controller' +
```

Comment:
Ummm, I don't think this is the right name here?

Reply:
Sorry again!